hyperparameter_tuning/xgboost_random_log/hpo_xgboost_random_log.ipynb

```diff
@@ -2,7 +2,6 @@
  "cells": [
```

Comment:
<p>Can we keep the plot and add it as image file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>